src/components/SpaceListItem/SecondLineElement/SecondLineElement.unit.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import SecondLineElement from '.';
+import DividerDot from '../../DividerDot';
+
+describe('<SecondLineElement />', () => {
+  const exampleChildProp = 'example-text';
+  const showDividerDot = true;
+
+  describe('snapshot', () => {
+    it('should match snapshot with children', () => {
+      expect.assertions(1);
+
+      const container = mount(<SecondLineElement>{exampleChildProp}</SecondLineElement>);
+
+      expect(container).toMatchSnapshot();
+    });
+
+    it('should match snapshot with children and showDividerDot', () => {
+      expect.assertions(1);
+
+      const container = mount(
+        <SecondLineElement showDividerDot={showDividerDot}>{exampleChildProp}</SecondLineElement>
+      );
+
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('attributes', () => {
+    it('should have provided children', async () => {
+      expect.assertions(1);
+
+      const container = mount(<SecondLineElement>{exampleChildProp}</SecondLineElement>)
+        .find('[data-test="second-line-element-child"]')
+        .getDOMNode();
+
+      expect(container.textContent).toBe(exampleChildProp);
```

Comment:
You can test text children without additional node like this:
```suggestion
      const container = mount(<SecondLineElement>{exampleChildProp}</SecondLineElement>);

      expect(container.text()).toEqual(exampleChildProp);
```

Reply:
That's much simpler, thank you! I've changed those unit tests and removed the unnecessary dom element from SecondLineElement.